iotedgehubdev/edgecert.py

```diff
@@ -1,44 +1,73 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License.
 
-
 from .certutils import EdgeCertUtil
 from .constants import EdgeConstants
+from .output import Output
 
 
 class EdgeCert(object):
     def __init__(self, certs_dir, hostname):
         self.certs_dir = certs_dir
         self.hostname = hostname
+        self.cert_util = EdgeCertUtil()
```

Comment:
not move to EdgeCert member. The cert_util has status. And the cert_chain should reset each time it used to generate the set of certs. The class may be refactored later. But based on the current implementation. Not put it as member.